src/Makefile.am

```diff
@@ -73,6 +75,8 @@ libproj_la_SOURCES = \
 	geocent.c geocent.h pj_utils.c pj_gridinfo.c pj_gridlist.c \
 	jniproj.c pj_mutex.c pj_initcache.c pj_apply_vgridshift.c geodesic.c \
 	pj_strtod.c
+	\
```

Comment:
> I don't know why Travis doesn't just give up earlier, though.

And the answer to that would be, [because it's commented out for some reason](https://github.com/OSGeo/proj.4/blame/master/autogen.sh#L35).